functionScripts.js

```diff
@@ -0,0 +1,97 @@
+const morseAlphabet = {
+  A: ".-",
+  B: "-...",
+  C: "-.-.",
+  D: "-..",
+  E: ".",
+  F: "..-.",
+  G: "--.",
+  H: "....",
+  I: "..",
+  J: ".---",
+  K: "-.-",
+  L: ".-..",
+  M: "--",
+  N: "-.",
+  O: "---",
+  P: ".--.",
+  Q: "--.-",
+  R: ".-.",
+  S: "...",
+  T: "-",
+  U: "..-",
+  W: ".--",
+  X: "-..-",
+  Y: "-.--",
+  Z: "--..",
+  " ": "/",
+  0: "-----",
+  1: ".----",
+  2: "..---",
+  3: "...--",
+  4: "....-",
+  5: ".....",
+  6: "-....",
+  7: "--...",
+  8: "---..",
+  9: "----.",
+  // ",": "--..--",
+  // ".": ".-.-.-",
+  // "?": "..--..",
+  // ";": "-.-.-.",
+  // ":": "---...",
+  // "/": "-..-.",
+  // "-": "-....-",
+  // "(": "-.--.",
+  // ")": "-.--.-",
+  // "=": "-...-",
+  // "+": ".-.-.",
+  // "@": ".--.-.",
+};
+
+export const checkNormOrMorse = (string) => {
+  if (string[0] === "." || string[0] === "-") {
+    return "Morse Code";
+  } else {
+    return "Normal Text";
+  }
+};
+
+export const invalidCharCheck = (string) => {
+  const StrToArr = string.split("");
+  const invalidChars = /^[a-zA-Z0-9-/.]*$/;
```

Comment:
This is picking up spaces as invalid chars as well. You could include whitespace as valid